science/exe.py

```diff
@@ -542,14 +542,15 @@ def dest_dir_option():
 
 def use_platform_suffix_option():
     return click.option(
-        "--use-platform-suffix",
-        is_flag=True,
+        "--use-platform-suffix/--no-use-platform-suffix",
```

Comment:
The docsite / this CLI help rendered here: https://jsirois.github.io/lift/cli.html#science-lift-build